### PR TITLE
GA blog post

### PR DESCRIPTION
**Pull request by KFishner**

### CODE REVIEW

File: source/blog/2015-07-07-atlas-general-availability.html.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+HashiCorp is now entering a new chapter. We have always been and will continue to be firmly committed to improving and supporting our open source projects; they are the foundation of our company and Atlas. Alongside our technological developments and community building, we are committed to building a scalable and customer-friendly business.
+
+READMORE
+
+## Journey from Tech Preview to General Availability
+
+When Atlas launched into tech preview, it could only host Vagrant boxes and run Packer builds that produced AMIs. Today, just seven months later, Atlas deeply integrates all of HashiCorp’s open source products: Vagrant, Packer, Terraform, Consul, and Vault. Atlas hosts Vagrant boxes and adds extra features on top of Vagrant. Atlas can run Packer builds resulting in any output format: VMware, AWS, Docker, VirtualBox, and more. Atlas plans and builds infrastructure with Terraform. Atlas monitors and configures launched hosts and services with Consul.
```
Comment (sethvargo) [inline source/blog/2015-07-07-atlas-general-availability.html.md]: Watch the "special quotes". There's a few `’` that should be `'`. It tends to look funny.

File: source/blog/2015-07-07-atlas-general-availability.html.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  * **Link GitHub and Terraform** - Terraform encourages infrastructure as code: define your infrastructure in versionable text files. GitHub is the best place to store and version the text files themselves. You can link GitHub to Atlas and Atlas will automatically run Terraform plans as part of pull reviews. This lets you see the effect of a code change on your infrastructure!
+
+To learn more about Terraform features and instructions for using them, [read the documentation](https://atlas.hashicorp.com/help/terraform/features).
+
+## Consul features in Atlas
+
+Consul is our tool for service discovery, configuration, and monitoring. Atlas integrates with Consul to provide features that help with bootstrapping Consul, visualizing cluster state, and alerting based on Consul cluster changes.
+
+![Consul in Atlas screenshot](/images/blog/atlas/consul-screenshot.png)
```
Comment (sethvargo) [inline source/blog/2015-07-07-atlas-general-availability.html.md]: I always forget if this works in prod. Let's just deploy and verify before tweeting.

**Comment by pearkes** [comment]
Just realized the tf screenshot is from a weird situation, going to update

**Comment by KFishner** [comment]
@pearkes updated

**Pull request merged**